ios/Demo-iOS/Sources/Views/SitePreparationView.swift

```diff
@@ -0,0 +1,330 @@
+
+    @State
+    private var viewModel: SitePreparationViewModel
+
+    init(site: ConfigurationItem) {
+        self.viewModel = SitePreparationViewModel(configurationItem: site)
+    }
+
+    var body: some View {
+        Group {
+            if let configuration = self.viewModel.editorConfiguration {
+                loadedView(configuration: configuration)
+            } else {
+                ProgressView("Loading Site Configuration")
+            }
+        }
+        .navigationTitle("Editor Configuration")
+        .onAppear {
+            self.viewModel.startLoading()
+        }
+    }
+
+    func loadedView(configuration: EditorConfiguration) -> some View {
+        Form {
+
+            if viewModel.editorDependencies != nil {
+                Text("Editor Dependencies Loaded – the editor should load instantly")
+            } else {
+                Text("Editor Dependencies Missing – the editor will need to load them when it starts")
+            }
+
+            Section("Feature Configuration") {
+                Toggle("Enable Native Inserter", isOn: $viewModel.enableNativeInserter)
+                Toggle("Enable Network Logging", isOn: $viewModel.enableNetworkLogging)
+
+                // TODO: Loading this from the server would allow us to validate Custom Post Type support
+                Picker("Post Type", selection: $viewModel.postType) {
+                    Text("Post").tag("post")
+                    Text("Page").tag("page")
+                }
+            }
+
+            if let editorConfiguration = viewModel.editorConfiguration {
+                Section("Editor Configuration Details") {
+                    KeyValueRow(key: "Site URL", value: editorConfiguration.siteURL.absoluteString)
+                    KeyValueRow(key: "API Root", value: editorConfiguration.siteApiRoot.absoluteString)
+                    KeyValueRow(key: "Supports Block Assets", value: editorConfiguration.shouldUsePlugins)
+                    KeyValueRow(key: "Supports Theme Styles", value: editorConfiguration.shouldUseThemeStyles)
+                }
+            }
+
+            self.preloadSection
+
+            if let error = viewModel.error {
+                Section("Error") {
+                    Text(error.localizedDescription)
+                }
+            }
+
+            Section {} footer: {
+                Button(action: {
+                    self.viewModel.buildAndLoadConfiguration(navigation: navigation)
+                }) {
+                    Spacer()
+                    Text("Start Editor")
+                    Spacer()
+                }.buttonStyle(.borderedProminent)
+            }
+        }
+    }
+
+    var preloadSection: some View {
+        Section {
+            Button("Prepare Editor") {
+                withAnimation {
+                    self.viewModel.prepareEditor()
+                }
+            }.disabled(self.viewModel.disableButtons)
+
+            Button("Prepare Editor ignoring cache") {
```

Comment:
Seems like iOS prefers titlecase. 
```suggestion
            Button("Prepare Editor Ignoring Cache") {
```